src/agent/onefuzz-telemetry/src/lib.rs

```diff
@@ -200,7 +200,7 @@ mod global {
     pub fn set_clients(instance: Option<TelemetryClient>, shared: Option<TelemetryClient>) {
         use Ordering::SeqCst;
 
-        let last_state = STATE.compare_and_swap(UNSET, SETTING, SeqCst);
+        let last_state = STATE.compare_exchange(UNSET, SETTING, SeqCst, SeqCst);
```

Comment:
This isn't a drop in replacement, the return types are different.

Reply:
Speculative `git push` (-;